tests/cephfs/cephfs_nfs/test_cephfs_nfs_ha_node_replacement.py

```diff
@@ -0,0 +1,367 @@
+        for i in interm_list:
+            fs_util_v1.wait_for_service_to_be_in_running(client1, i)
+        haproxy_ls_after_step_2 = fs_util_v1.get_deamon_status(client1, "haproxy")
+        keepalived_ls_after_step_2 = fs_util_v1.get_deamon_status(client1, "keepalived")
+        haproxy_ls_after_hostnames_step_2 = [
+            entry["hostname"] for entry in haproxy_ls_after_step_2
+        ]
+        keepalived_ls_after_hostnames_step_2 = [
+            entry["hostname"] for entry in keepalived_ls_after_step_2
+        ]
+        if (
+            haproxy_ls_after_hostnames_step_2 == haproxy_ls_after_hostnames
+            or keepalived_ls_after_hostnames == keepalived_ls_after_hostnames_step_2
+        ):
+            raise CommandFailed(
+                f"The ha proxy node did not change even after power off ."
+                f"\nPrevious haproxy nodes: {haproxy_ls_after_hostnames}"
+                f"\nCurrent haproxy nodes: {haproxy_ls_after_hostnames_step_2}"
+                f"\nPrevious keepalived nodes: {keepalived_ls_after_hostnames}"
+                f"\nCurrent keepalived nodes: {keepalived_ls_after_hostnames_step_2}"
+                f"\nHA Proxy services not started BZ : 2240258"
+            )
+        if nfs_servers[3].node.hostname not in haproxy_ls_after_hostnames_step_2:
+            raise CommandFailed("HA Proxy services not started BZ : 2240258")
+        if nfs_servers[3].node.hostname not in keepalived_ls_after_hostnames_step_2:
+            raise CommandFailed("Keepalived services not started BZ : 2240258")
+        log.info(
+            f"The active node changed after Power off."
+            f"\nPrevious active node: {active_nfs.node.hostname}"
+            f"\nCurrent active node: {active_nfs_after.node.hostname}"
+        )
+        active_nfs_ipmi = IPMIPowerControl(
+            host=f"{backend_server[0].node.hostname}.ipmi.ceph.redhat.com",
+            username="inktank",
+            password="ApGNXcA7",
+        )
+        active_nfs_ipmi.power_up()
+        fs_util_v1.wait_for_host_online(client1, backend_server[0])
+        ps_ls = client1.exec_command(
+            sudo=True, cmd=f"ceph orch ps {backend_server[0].node.hostname} -f json"
+        )
+        if "nfs" in ps_ls:
+            raise CommandFailed(
+                f"Still Holding the nfs services on the removed server {backend_server[0].node.hostname}"
+            )
+
+        backend_server = get_active_nfs_server(client1, nfs_name, ceph_cluster)
+        for server in backend_server:
+            log.info(f"Active backend servers after reboots {server.node.hostname}")
+        log.info("Test completed successfully")
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Cleaning Up")
+        client1.exec_command(
+            sudo=True, cmd=f"rm -rf {nfs_mounting_dir}*", check_ec=False
```

Comment:
we need to remove both directories created and appended using variable - "mount_dir". Using "nfs_mounting_dir" is only removing the last directory created. please check logs.